doc/src/fix_ave_moments.rst

```diff
@@ -0,0 +1,286 @@
+specified values may represent calculations performed by computes and
+fixes which store their own "group" definitions.
+
+Each listed value can be the result of a :doc:`compute <compute>` or
+:doc:`fix <fix>` or the evaluation of an equal-style or vector-style
+:doc:`variable <variable>`.  In each case, the compute, fix, or variable
+must produce a global quantity, quantity, not a per-atom or local
+quantity.  If you wish to spatial- or time-average or histogram per-atom
+quantities from a compute, fix, or variable, then see the :doc:`fix
+ave/chunk <fix_ave_chunk>`, :doc:`fix ave/atom <fix_ave_atom>`, or
+:doc:`fix ave/histo <fix_ave_histo>` commands.  If you wish to sum a
+per-atom quantity into a single global quantity, see the :doc:`compute
+reduce <compute_reduce>` command.
+
+:doc:`Computes <compute>` that produce global quantities are those which
+do not have the word *atom* in their style name.  Only a few :doc:`fixes
+<fix>` produce global quantities.  See the doc pages for individual
+fixes for info on which ones produce such values.  :doc:`Variables
+<variable>` of style *equal* and *vector* are the only ones that can be
+used with this fix.  Variables of style *atom* cannot be used, since
+they produce per-atom values.
+
+The input values must all be scalars or vectors with a bracketed term
+appended, indicating the :math:`I^\text{th}` value of the vector is
+used.
+
+The result of this fix can be accessed as a vector, containing the
+interleaved moments of each input in order.  The first requested moment
+of input 1 has index 1, the second index 2, the first of input 2 has
+index 3 and so on.
+
+----------
+
+For input values from a compute or fix or variable, the bracketed index
+I can be specified using a wildcard asterisk with the index to
+effectively specify multiple values.  This takes the form "\*" or "\*n"
+or "m\*" or "m\*n".  If :math:`N` is the size of the vector, then an
+asterisk with no numeric values means all indices from 1 to :math:`N`.
+A leading asterisk means all indices from 1 to n (inclusive).  A
+trailing asterisk means all indices from n to :math:`N` (inclusive).  A
+middle asterisk means all indices from m to n (inclusive).
+
+Using a wildcard is the same as if the individual elements of the vector
+or cells of the array had been listed one by one.  For examples, see the
+description of this capability in :doc:`fix ave/time <fix_ave_time>`.
+
+----------
+
+The :math:`N_\text{every}`, :math:`N_\text{repeat}`, and
+:math:`N_\text{freq}` arguments specify on what time steps the input
+values will be used in order to contribute to the average.  The final
+statistics are generated on time steps that are a multiple of
+:math:`N_\text{freq}`\ .  The average is over a window of up to
+:math:`N_\text{repeat}` quantities, computed in the preceding portion of
+the simulation every :math:`N_\text{every}` time steps.
+
+.. note::
+
+    Contrary to some fix ave/* commands, the values of this fix are not
```

Comment:
Contrary to most fix ave/* commands, it is not required that Nevery * Nrepeat <= Nfreq.  This is to allow the user to choose the time window and number of samples contributing to the output at each Nfreq interval.